apiserver/pkg/util/cluster.go

```diff
@@ -473,7 +473,7 @@ func buildWorkerPodTemplate(imageVersion string, envs *api.EnvironmentVariables,
 							ValueFrom: &corev1.EnvVarSource{
 								ResourceFieldRef: &corev1.ResourceFieldSelector{
 									ContainerName: "ray-worker",
-									Resource:      "limits.cpu",
+									Resource:      "limits.memory",
```

Comment:
Something like this should have been caught by unit tests. Is it possible to add a unit test to prevent this type of bug in the future?

Reply:
add test into unit tests.